src/components/AddBook.js

```diff
@@ -44,6 +44,6 @@ export default function AddBook() {
         </select>
         <button type="submit">Add Book</button>
```

Comment:
- [ ] Kindly make sure also this add button has a white color.

![image](https://user-images.githubusercontent.com/65068771/162338049-b6ff838c-c1d3-4611-8255-65114827e724.png)